examples/js/renderers/CSS3DRenderer.js

```diff
@@ -278,13 +280,18 @@ THREE.CSS3DRenderer = function () {
 		var cameraCSSMatrix = 'translateZ(' + fov + 'px)' +
 			getCameraCSSMatrix( camera.matrixWorldInverse );
 
+		if ( camera.type === 'OrthographicCamera' ) {
+
+			cameraCSSMatrix = 'scale(' + fov + ')' + cameraCSSMatrix;
```

Comment:
I _think_ this simpler formula will work.
```javascript
if ( camera.type === 'OrthographicCamera' ) {

	cameraCSSMatrix = 'scale(' + fov + ')' + getCameraCSSMatrix( camera.matrixWorldInverse );

} else {

	cameraCSSMatrix = 'translateZ(' + fov + 'px)' + getCameraCSSMatrix( camera.matrixWorldInverse );

}
```

Reply:
It works! Thank you for your suggestion.
Should we use ternary operator, or is it supposed to done in `if` condition?

```javascript
var cameraCSSMatrix = camera.type === 'OrthographicCamera' ?
	'scale(' + fov + ')' + getCameraCSSMatrix( camera.matrixWorldInverse ) :
	'translateZ(' + fov + 'px)' + getCameraCSSMatrix( camera.matrixWorldInverse );
```